atomicapp/cli/main.py

```diff
@@ -188,12 +209,12 @@ def set_arguments(self):
 
         parser_install.add_argument(
             "--destination",
-            dest="target_path",
+            dest="destination",
             default=None,
             help="Destination directory for install")
 
         parser_install.add_argument(
-            "APP",
+            "app_spec",
```

Comment:
this PR is already merged.. we can update to app_src in new PR in the future if you want.